### ISSUE TITLE
Setting Accessibility Text for Images in ImageResponse and CardResponse

### ISSUE THREAD

**Issue by jzxhuang** [dialogflow/dialogflow-fulfillment-nodejs#10]
**Issue:** There' no way to set the accessibility text for images in ImageResponse and CardResponse objects. Accessibility text is required for images on AoG, currently I see the text 'accessibility text' is used as filler. 

This should be a pretty straightforward addition, just running it by you to get an OK - I do have one main question. Should accessibilityText be:
- A mandatory parameter the user must provide anytime an image is used and the response platform is AoG
- Or an optional parameter, with default text used if the user does not specify accessibilityText.

I guess it's more product-related than an implementation question, and from using DialogFlow/AoG I know accessibility text **must** be set any time there's an image... so I think this library should reflect that? 

Anyways, brief outline of the changes that would be made:

1.  **ImageResponse Object:** 
  Add a method setAccessibilityText(accessibilityText).
  Modify constructor, if an object is passed in, look for an accessibilityText key.
  Update getV1/V2Object methods appropriately for AoG platform

2. **CardResponse Object:**
This one is a bit trickier, I don't think it makes sense to add another method setImageAccessibilityText() since accessibilityText is only used for AoG and dependent on if you have an image in your card in the first place. 
I think the better approach is to allow the user to pass in an object in setImage(). They can specify the accessibility text there. getV1/V2Object methods would be updated appropriately for AoG platform.

**Comment by matthewayne** [comment]
The builtin responses (including cards and images you mention) are meant to support cross platform development.  If you'd like to access platform specific features please use the Actions on Google client library integration or custom payload.  For instance, using the Actions on Google library integration:
```    
const {WebhookClient} = require('dialogflow-fulfillment');
const {  BasicCard } = require('actions-on-google') //use v2 alpha

const agent = new WebhookClient({ req, res });
let conv = agent.conv();
conv.ask(new BasicCard({
  title: 'Card Title',
  image: { // Mostly, provide anonymous Objects with the properties defined by the raw JSON API
    url: 'https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png',
    accessibilityText: 'Google Logo',
  },
  // Some complex objects have class wrappers but can use raw API when needed
  buttons: new Button({
    title: 'Button Title',
    url: 'https://www.google.com',
  }),
  display: 'WHITE',
}));
agent.add(conv);
```

or using the custom payload integration:

```
const {WebhookClient, Payload} = require('dialogflow-fulfillment');
const agent = new WebhookClient({ req, res });
agent.add(new Payload( agent.ACTIONS_ON_GOOGLE, {
  "google": {
    "expectUserResponse": true,
    "richResponse": {
      "items": [
        {
          "simpleResponse": {
            "textToSpeech": "This is a Basic Card:"
          }
        },
        {
          "basicCard": {
            "title": "Card Title",
            "image": {
              "url": "https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png",
              "accessibilityText": "Google Logo"
            },
            "buttons": [
              {
                "title": "Button Title",
                "openUrlAction": {
                  "url": "https://www.google.com"
                }
              }
            ],
            "imageDisplayOptions": "WHITE"
          }
        }
      ]
    }
  }
})
```

**Comment by harmnot** [comment]
I follow those but not working, it not showing off on web demo 
and when I use new card from dialogflow-fulfillment on my console diaglogflow it was working, but when on web demo it was  empty response ? why ?

**Comment by Gouraviitk** [comment]
> I follow those but not working, it not showing off on web demo
> and when I use new card from dialogflow-fulfillment on my console diaglogflow it was working, but when on web demo it was empty response ? why ?

I am facing the exact same issue
